images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -16,7 +16,12 @@ function Get-SqliteVersion {
 }
 
 function Get-MySQLVersion {
-    $mySQLVersion = mysqld --version | Take-OutputPart -Part 2
+    if (Test-IsUbuntu20) {
+        $mySQLVersion = mysqld --version | Take-OutputPart -Part 2
+    }
+    else {
+        $mySQLVersion = mysqld --version | Take-OutputPart -Part 2 | Take-OutputPart -Part 0 -Delimiter "-"
+    }
```

Comment:
Maybe something like this?

```suggestion
    $mySQLVersion = mysqld --version | Take-OutputPart -Part 2
    if (-not (Test-IsUbuntu20)) {
        $mySQLVersion = $mySQLVersion | Take-OutputPart -Part 0 -Delimiter "-"
    }
```

Reply:
I would change to  if (Test-IsUbuntu18) 😃